### PR TITLE
Fp16 NHWC Max and Average Pooling

### PR DESCRIPTION
**Pull request by chenfucn**
### Description
Max and average pooling operators for fp16, NHWC 

### Motivation and Context
Continue on the steps for fp16 inference support

### CODE REVIEW

**Review by jchen351 — APPROVED** [review]

File: onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc [hunk 1]
```diff
@@ -45,6 +45,27 @@ ONNX_MS_OPERATOR_SET_SCHEMA(NhwcMaxPool, 1,
                                   ::onnxruntime::contrib::convPoolShapeInferenceNhwc(ctx, true, true, 0, 1);
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(NhwcAvgPool, 1,
```
Comment (yufenglee) [inline onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc:48]: >NhwcAvgPool [](http://example.com/codeflow?start=28&length=11)

what is the reason to not leverage the internal NHWC ops

Comment (chenfucn) [reply onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc:48]: Can't get it to work :-( would you provide some guidance?

**Review by snnn — APPROVED** [review]

**Review by yufenglee — APPROVED** [review]
:shipit:

**Pull request merged**